src/scss/base/index.scss

```diff
@@ -31,5 +32,5 @@
 
 [type=checkbox]:checked+label:before, [type=checkbox]:checked:disabled+label:before {
   background: #b1040c;
-//url(../../assets/img/check.png);
+  //url(../../assets/img/check.png);
```

Comment:
Please remove commented out code if you don't plan to use it.

Reply:
This existed before, are we okay to remove it?